app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -47,7 +47,7 @@ class MainActivity : AppCompatActivity(), OnMediaOptionSelectedListener, OnReque
         private val UNDERLINE = "<u>Underline</u><br>"
         private val STRIKETHROUGH = "<s class=\"test\">Strikethrough</s><br>" // <s> or <strike> or <del>
         private val ORDERED = "<ol><li>Ordered</li></ol>"
```

Comment:
Adding an empty list item to the ordered list might help illustrate the difference in lists.